### ISSUE TITLE
EKS - Reflect Addon status to CAPI

### ISSUE THREAD

**Issue by mproffitt** [giantswarm/roadmap#3129]
EKS cluster import currently does not recognise EKS Addons leading to a situation whereby Cluster API attempts to remove addons that are installed in the adopted/imported cluster.

This behaviour is incorrect and the addons should be left in place unless specifically marked for removal.

As crossplane does not return a list of installed addons as part of cluster lookup, this most likely requires an additional API check to recover a list of addons installed in the cluster and for this to be added to the XR status for later patching.

- [ ] Discovery - best method of capturing installed addon details
- [ ] Implementation
- [ ] Testing

**Comment by weatherhog** [comment]
@mproffitt is this still needed?

**Comment by mproffitt** [comment]
Yes this one is still required for when we come back to EKS adoption

**Comment by mproffitt** [comment]
No longer required, closing
